data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -1239,8 +1242,7 @@
       [ "mutagen", 1 ]
     ],
     [
-      [ "meat_tainted", 3 ],
-      [ "dry_meat_tainted", 3 ],
+      [ "meat_insect_tainted", 3 ],
```

Comment:
How do bug guts make you a squid person? Mullusca and Athropoda are not remotely the same.